### DISCUSSION TITLE
How to identify an actor in Physx

### DISCUSSION THREAD

**Discussion by carlosedubarreto** [NVIDIA-Omniverse/PhysX#31] [Q&A] [answered]
Im searching how to do this but im not finding.

I would like to know if there is a way to set or to get an id of each actor in the scene.

My intention is to create a scene with the simulation and have some sort of id to link it with another scene in other software.

Thanks.

**Comment by preist-nvidia** [comment, marked as answer]
You have two options:

userData: https://nvidia-omniverse.github.io/PhysX/physx/5.1.0/_build/physx/latest/class_px_actor.html?highlight=userdata#_CPPv4N7PxActor8userDataE

or a name: https://nvidia-omniverse.github.io/PhysX/physx/5.1.0/_build/physx/latest/class_px_actor.html?highlight=setname#_CPPv4N7PxActor7setNameEPKc

**Reply by carlosedubarreto** [reply]
Thnaks a lot for the info, I'll try it.

**Reply by carlosedubarreto** [reply]
Found this snoppet that can help a lot
https://github.com/NVIDIA-Omniverse/PhysX/blob/85fd98157dcb7df3cfc26fa9c07c6052c320b92f/physx/snippets/snippettriggers/SnippetTriggers.cpp
